cassandra/cassandra-prestop.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+echo "About to call the nodetool drain command"
```

Comment:
It doesn't have much value other than providing some output from the hook which could be outputted to the event.